website/static/css/fangorn.css

```diff
@@ -37,6 +37,10 @@ color: #F00;
 text-align: center;
 }
 
+#treeGrid span {
```

Comment:
@sloria The reason that I used the ID was actually specifically to make sure that not all grids used these CSS rules. For example, the project organizer is a very similar grid, but due to some other unique CSS based on its ID, these changes would make it look really bad. `#treeGrid` was the only thing I could find that only appeared on the grids I wanted to change.

If you would like me to, however, I could create a new class or ID to put these changes under, then add them where the changes need to go into effect. It would just mean that anyone creating another similar file viewer somewhere would need to remember my ID/class as well.